projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/ProjectorStatusWidget.kt

```diff
@@ -96,23 +91,45 @@ class ProjectorStatusWidget(project: Project)
     }
   }
 
-  private fun updateText() = "Projector"
+  private fun updateText() = "Projector" //  (${clientsCount()})"
 
   private fun updateTooltip(): String {
     return when {
       isActivationNeeded() -> "Activation is needed"
-      isProjectorRunning() -> "Projector is running"
+      isProjectorRunning() -> {
+        "Projector is running" //. Connected client(s): ${clientsCount()}"
+      }
       isProjectorAutoStarting() -> "Projector is starting"
-      isProjectorDisabled() -> "Projector is disabled"
+      isProjectorDisabled() -> "Projector is down"
```

Comment:
Since we've dropped a pop-up step, let's make these tooltips more descriptive so it's clear that a click is supported. I suggest something like "Projector is running, click to see and change current session settings". And the same for other text in states that support a click.

Also, why have you changed `disabled` to `down`? The latter one seems to be not quite clear, it's more like `Projector is crashed` to me 🤣

Reply:
Fixed